src/app/content/components/__snapshots__/BookBanner.spec.tsx.snap

```diff
@@ -90,25 +90,12 @@ Array [
     <div
       className="BookBanner__TopBar-sc-1avy0c0-1 jVswHX"
     >
-      <a
-        className="BookBanner__BookTitleLink-sc-1avy0c0-3 iGPDns"
-        data-testid="details-link-expanded"
-        href="/error/404"
-        onClick={[Function]}
+      <span
+        className="BookBanner__BookTitle-sc-1avy0c0-2 gUeKV"
+        data-testid="book-title-expanded"
```

Comment:
These changes are intended, right?

Reply:
yea i fussed with the bookbanner because there were two variables being used in conditions that were actually associated so i combined the logic, i think its more reliable now